Refresh: update Cloud free trial doc for new flow (#36)
crate/cloud-getting-started: docs/free-trial.rst

==========

If you want to try out CrateDB Cloud's potential, there are multiple ways to do

> this. One is to simply subscribe to CrateDB Cloud, by following our direct
> subscription tutorial, and to choose the Development subscription plan. As a
> secondary option, you can also refer to our SaaS marketplace offers on `Azure
> Marketplace`_ and `AWS Marketplace`_ and choose the Development plan there.
> (For more information on what that means, see our `reference on subscription
> plans`_).
> 
> A different approach is to avail yourself of the possibility of our free trial:
> a fully functioning CrateDB Cloud cluster running for 14 days without cost, so
> you can experience the benefits of CrateDB Cloud without any immediate
> commitment.

Reviewer: ```suggestion
this. One is to subscribe to CrateDB Cloud by following our direct
subscription tutorial, and to choose the Development subscription plan.
Another option is via our SaaS marketplace offers on `Azure Marketplace`_ 
and `AWS Marketplace`_ and choose the Development plan there. For more
information on the differences, see our `reference on subscription plans`_.

A third option is our free trial: a fully functioning CrateDB Cloud cluster
running for 14 days without cost, so you can experience the benefits of
CrateDB Cloud without any immediate commitment.
```

Sign up
=======

> In order to access the free trial, you must first sign up with an account.
> The signup process starts at the webpage form. Simply fill out the form and
> click *Submit* to proceed.

Reviewer: ```suggestion
In order to access the free trial, you must first sign up with an account.
The signup process starts at the webpage form. Fill out the form and
click *Submit* to proceed.
```

.. image:: _assets/img/free-trial-webform.png
   :alt: Free trial webform

> You will see a forwarding screen confirming your information was received. This
> will forward you to the signup process for creating a username and password for
> CrateDB Cloud (supported by Cognito). Follow the steps and log in (or if you
> already have a username and password for CrateDB Cloud, simply log in now).
> When you have successfully logged in, you will be forwarded to the CrateDB
> Cloud Console subscription overview. It is from here that you can make use of
> the free trial option. The first step is to set up an organization.

Reviewer: ```suggestion
You will see a forwarding screen confirming your information was received. This
takes you to the signup process where you can create a username and password for
CrateDB Cloud (supported by Cognito). Follow the steps and log in (or if you
already have a username and password for CrateDB Cloud, log in with those credentials).
Once logged in, you will be forwarded to the CrateDB Cloud Console subscription overview.
where you will find the free trial option. The first step is to set up an organization.
```
